assets/src/edit-story/app/media/utils/useFFmpeg.js

```diff
@@ -32,6 +32,11 @@ import {
 } from '../../../constants';
 import getFileName from './getFileName';
 
+export const VIDEO_SIZE_THRESHOLD = {
```

Comment:
this was a good idea, thanks for cleaning this up to be shared. 

if i'm reading Will's code update right in #7335 we want to be checking for `720 x 1280` as the min now instead of `1080 x 1920`?

Reply:
That's right, just merged the change.

Reply:
Oh awesome it got merged. I'll update. I just didn't wanna make the change to the size before he merged it in. just wanted to set it up to share the values. I can update now tho since it got merged